contactql/es/testdata/to_query.json

```diff
@@ -1735,6 +1735,50 @@
             }
         }
     },
+    {
+        "description": "history equality",
+        "query": "history = \"registration\"",
+        "elastic": {
+            "term": {
+                "flow_history_ids": 234
+            }
+        }
+    },
+    {
+        "description": "flow inequality",
+        "query": "history != \"registration\"",
+        "elastic": {
+            "bool": {
+                "must_not": {
+                    "term": {
+                        "flow_history_ids": 234
+                    }
+                }
+            }
+        }
+    },
+    {
+        "description": "history is set",
+        "query": "history != \"\"",
+        "elastic": {
+            "exists": {
+                "field": "flow_history_ids"
+            }
+        }
+    },
+    {
+        "description": "history is not set",
+        "query": "history = \"\"",
```

Comment:
This means the contact never been in any flow?

Reply:
How does archived runs affect the flow history?

Reply:
1. Correct
2. Yeah.. flow history = 90 days of flow history... as it currently does for flow starts